javase/pom.xml

```diff
@@ -31,6 +31,11 @@
       <artifactId>jcommander</artifactId>
       <version>1.48</version>
     </dependency>
+    <dependency>
+      <groupId>net.imagej</groupId>
+      <artifactId>ij</artifactId>
+      <version>1.48r</version>
```

Comment:
I was using a older version, therefore I have put the first one, but yeah it is 1.50b